exec/java-exec/src/test/java/org/apache/drill/test/ClusterFixture.java

```diff
@@ -471,6 +471,21 @@ public void close() throws Exception {
   }
 
   /**
+   * Shutdown the drillbit given the name of the drillbit.
+   */
+  public void closeDrillbit(final String drillbitName) throws Exception {
+    Exception ex = null;
+    for (Drillbit bit : drillbits()) {
+      if(bit.equals(bits.get(drillbitName))) {
+        bit.close();
+      }
+    }
+    if(ex != null) {
```

Comment:
Looks like `ex` is declared, but never set. Was the goal to capture exceptions from `bit.close()`?